application/helpers/export_helper.php

```diff
@@ -1341,6 +1341,29 @@ function quexml_set_default_value(&$element, $iResponseID, $qid, $iSurveyID, $fi
     }
 }
 
+/**
+ * Format defaultValue of Date/Time questions according to question date format
+ *
+ * @param mixed $element DOM element with the date to change
+ * @param int $qid The qid of the question
+ * @param int $iSurveyID The survey id
```

Comment:
I prefer `@return void`, so it doesn't look like it's forgotten. But I don't think it's mandated by PSR-12.

Reply:
Fixed